src/arche/readers/items.py

```diff
@@ -61,7 +61,8 @@ def __init__(
         self._limit = None
         self.filters = filters
         self.expand = expand
-        df = self.fetch_data()
+        self.raw = self.fetch_data()
+        df = pd.DataFrame(list(self.raw))
```

Comment:
We need this cast, otherwise pandas will put all dicts in one column.

Reply:
I'll check it